pkg/errors/errors.go

```diff
@@ -108,7 +108,7 @@ var (
 
 	// meta related errors
 	ErrMetaNewClientFail    = errors.Normalize("create meta client fail", errors.RFCCodeText("DFLOW:ErrMetaNewClientFail"))
-	ErrMetaOpFail           = errors.Normalize("meta operation fail:%v", errors.RFCCodeText("DFLOW:ErrMetaOpFail"))
+	ErrMetaOpFail           = errors.Normalize("meta operation fail", errors.RFCCodeText("DFLOW:ErrMetaOpFail"))
```

Comment:
Found some `ErrMetaOpFail` is used as `return cerrors.ErrMetaOpFail.GenWithStackByArgs("input resource meta is nil")`, which will generate an error like
```go
"[DFLOW:ErrMetaOpFail]meta operation fail%!(EXTRA string=input project info is nil)"
```